File: mainHerencia.py
from Herencia import*

def main():
    print("RESTA")
    resultado=ClaseResta(50,25,25)
    print(resultado.getRestas())

    print("SUMA")
    resultado2=SumaResultad(50,50,100)
    print(resultado2.getSumas())

    print("DIVISION")
    resultado3=DivisionResultado(10,2,5)
    print(resultado3.getDivisiones())


if __name__=="__main__":
    main()

File: mainHerencia.java
package Herencia;

public class mainHerencia {
    public static  void main(String[]args){
        System.out.println("SUMA");
        ClaseSuma resultado = new ClaseSuma(45,34,34);
        System.out.println(resultado.getSonIguales());

        System.out.println("DIVISION");
        ClaseDivision resultado2 = new ClaseDivision(45,35,35);
        System.out.println(resultado2.getSonIguales());

        System.out.println("RESTA");
        ClaseResta resultado3 = new ClaseResta(50,25,25);
        System.out.println((resultado3.sonIguales));

    }
}
